cli/fathom_web/test/test_extract.py

```diff
@@ -0,0 +1,32 @@
+from ..commands.extract import BASE64_DATA_PATTERN
+
+
+def test_common_example():
+    """Confirm we handle to well behaving case"""
+    mime_type = 'image/png'
+    base64_string = 'aorienstar/tar/ararnsoine98daQAAAIST+++/rstienf='
+    test_string = f'data:{mime_type};base64,{base64_string}'
+    matches = list(BASE64_DATA_PATTERN.finditer(test_string))
```

Comment:
Can we call findall() instead and save a list() call?

Reply:
`findall()` does not return `Match` objects, while `finditer()` does. Since we use the `Match` objects in `fathom-extract`, we should test with `Match` objects as well.

Reply:
Good point! I didn't see that in my quick read.